### PR TITLE
Typo fix

### PR DESCRIPTION
**Pull request by cdmihai**

### CODE REVIEW

**Comment by RaulPerez1** [comment]
>         <xs:documentation>Suffix that can indicate additional information at the end of the verion (i.e. beta)</xs:documentation>

Typo here as well

---
Refers to: src/MSBuild/Microsoft.Build.CommonTypes.xsd:1804 in c2aac57. [](commit_id = c2aac578afb659b6e83a2ec5d632b506e67e6efa, deletion_comment = False)

File: src/MSBuild/Microsoft.Build.CommonTypes.xsd [hunk 1]
```diff
@@ -1796,7 +1796,7 @@ elementFormDefault="qualified">
     </xs:element>
     <xs:element name="VersionPrefix" type="msb:StringPropertyType" substitutionGroup="msb:Property" >
       <xs:annotation>
-        <xs:documentation><!-- _locID_text="VersionPrefix" _locComment="" -->Prefix that can indicate additional information at the end of the verion (i.e. beta)</xs:documentation>
+        <xs:documentation><!-- _locID_text="VersionPrefix" _locComment="" -->Prefix that can indicate additional information at the start of the version (i.e. beta)</xs:documentation>
```
Comment (RaulPerez1) [inline src/MSBuild/Microsoft.Build.CommonTypes.xsd]: >i.e. [](start = 154, length = 4)

These are supposed to be e.g. Same goes for the one below

**Comment by RaulPerez1** [comment]
:shipit:

File: src/MSBuild/Microsoft.Build.CommonTypes.xsd (hunk 1 above)
Comment (rainersigwald) [inline src/MSBuild/Microsoft.Build.CommonTypes.xsd]: Looking at how this is used, I'm not sure this is a good description.

```xml
    <VersionPrefix Condition=" '$(VersionPrefix)' == '' ">1.0.0</VersionPrefix>
    <Version Condition=" '$(VersionSuffix)' != '' ">$(VersionPrefix)-$(VersionSuffix)</Version>
    <Version Condition=" '$(Version)' == '' ">$(VersionPrefix)</Version>
```

So maybe this should be `Initial part of the version number, for example 1.0.0` ?

@nguerrera @eerhardt y'all seem to have been the ones to touch this in the SDK, do you have opinions?

Comment (eerhardt) [inline src/MSBuild/Microsoft.Build.CommonTypes.xsd]: Agreed this is not a good description for this property.

This is the best documentation I can find that explains these properties:  https://github.com/dotnet/cli/issues/5573#issuecomment-277377729

Comment (cdmihai) [inline src/MSBuild/Microsoft.Build.CommonTypes.xsd]: How about:

- VersionPrefix: When `Version` is not specified, `VersionPrefix` represents the first fragment of the version number (e.g. 1.0.0). The format is <VersionPrefix>[-<VersionSuffix>]
- VersionSuffix: Optional. When `Version` is not specified, `VersionSuffix` represents the second fragment of the version number (e.g. beta). The format is <VersionPrefix>[-<VersionSuffix>]

Comment (rainersigwald) [inline src/MSBuild/Microsoft.Build.CommonTypes.xsd]: Seems reasonable to me . . .

Comment (eerhardt) [inline src/MSBuild/Microsoft.Build.CommonTypes.xsd]: 👍

**Comment by cdmihai** [comment]
@RaulPerez1 we changed the description for `VersionSuffix` and `VersionPrefix`. Still look good to you?

**Comment by RaulPerez1** [comment]
Looks good to me.

**Comment by cdmihai** [comment]
@RaulPerez1 
Oops, I had forgotten to commit my changes when I pinged you. What about now? :)

File: src/MSBuild/Microsoft.Build.CommonTypes.xsd [hunk 2]
```diff
@@ -1796,12 +1796,12 @@ elementFormDefault="qualified">
     </xs:element>
     <xs:element name="VersionPrefix" type="msb:StringPropertyType" substitutionGroup="msb:Property" >
       <xs:annotation>
-        <xs:documentation><!-- _locID_text="VersionPrefix" _locComment="" -->Prefix that can indicate additional information at the end of the verion (i.e. beta)</xs:documentation>
+        <xs:documentation><!-- _locID_text="VersionPrefix" _locComment="" --> When Version is not specified, VersionPrefix represents the first fragment of the version string (e.g. 1.0.0). The syntax is VersionPrefix[-VersionSuffix].</xs:documentation>
```
Comment (rainersigwald) [inline src/MSBuild/Microsoft.Build.CommonTypes.xsd]: nit: leading space.

**Comment by RaulPerez1** [comment]
Thumbs up!

**Pull request merged**